src/Symfony/Validator/Metadata/Property/Restriction/PropertySchemaOneOfRestriction.php

```diff
@@ -42,7 +42,7 @@ public function __construct(iterable $restrictionsMetadata = [])
      */
     public function create(Constraint $constraint, ApiProperty $propertyMetadata): array
     {
-        $oneOfConstraints = $constraint->getNestedContraints();
+        $oneOfConstraints = method_exists($constraint, 'getNestedContraints') ? $constraint->getNestedContraints() : $constraint->constraints;
```

Comment:
```suggestion
        $oneOfConstraints = method_exists($constraint, 'getNestedContraints') ? $constraint->getNestedContraints() : $constraint->getNestedConstraints();
```

Why not like this?

Reply:
some constraints don't seem to have this method, not really sure *how*, but tests fail when using the nested constraints method.